Module.php

```
@@ -0,0 +1,209 @@
+        }
+        foreach ($adminJs as $p) {
+            $u = $basePath($p);
+            $sep = (strpos($u, '?') !== false) ? '&' : '?';
+            $view->headScript()->appendFile($u . $sep . $cb);
+        }
+    }
+
+    /**
+     * Handle the configuration form submission
+     */
+    public function handleConfigForm(Form $form, array $params, ViewModel $view)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$view'. (Unused Code Rules)

(UnusedFormalParameter)